lib/ueberauth.ex

```diff
@@ -69,7 +69,7 @@ defmodule Ueberauth do
   In phoenix setup a pipeline:
 
       pipeline :ueberauth do
-        Ueberauth.plug "/auth"
+        plug Ueberauth, base_path: "/auth"
```

Comment:
In that case, you could ditch the base_path option and rely on the conn.path_info/conn.request_path directly:


```elixir
scope "/" do
  pipe_through [:browser, :ueberauth]
  forward "/auth", Ueberauth
end
```